question-generation/main.py

```diff
@@ -0,0 +1,49 @@
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from pydantic import BaseModel
+from typing import List
+
+from nlp import get_questions
+
+app = FastAPI()
+
+# May risk unauthorized access
+origins = [
+    "*",
+]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+# JSON-like object
+class Data(BaseModel):
```

Comment:
Sophia mentioned that she likes handling JSON directly, at least during development phase (since it would allow any data format she wants to experiment with). So, I am considering changing this back to JSON instead of using pydantic. Any thoughts about this?